GageStatsServices/Startup.cs

```diff
@@ -129,22 +131,23 @@ public void ConfigureServices(IServiceCollection services)
                 options.ModelMetadataDetailsProviders.Add(new SuppressChildValidationMetadataProvider(typeof(Point)));
                 options.ModelMetadataDetailsProviders.Add(new SuppressChildValidationMetadataProvider(typeof(Geometry)));
             })                               
-                .AddJsonOptions(options => loadJsonOptions(options));                                                
+                .AddNewtonsoftJson(options => loadJsonOptions(options));                                                
```

Comment:
Just an FYI (I can't recall if we discussed this before). NewtonsoftJSON is now a dependency json serializer/deserializer, (it use to be imbedded in sys dotnetcore). and it is really good, and has been around for a long time. Recently, MS started creating their own, its fairly new, but might be worth keeping an eye on it, since it'll probably become the defacto. [Here ](https://michaelscodingspot.com/the-battle-of-c-to-json-serializers-in-net-core-3/#comments)is a link explaining some of the top contenders

Reply:
Thanks for the info!